circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/NavigableCircuitContent.kt

```diff
@@ -221,9 +217,19 @@ public object NavigatorDefaults {
   // Mirrors the forward and backward transitions of activities in Android 34
   public object DefaultDecoration : NavDecoration {
 
-    private val forward: ContentTransform = computeTransition(1)
+    /**
+     * The [ContentTransform] used for 'forward' navigation changes (i.e. items added to stack).
+     * This isn't meant for public consumption, so be aware that this may be removed/changed at any
+     * time.
+     */
+    public val forward: ContentTransform by lazy { computeTransition(1) }
```

Comment:
I wasn't sure on making these public, but I didn't want to copy everything over into the Android GestureNavDecoration impl.